### ISSUE TITLE
Next feature release?

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#4728]
Last feature release was on 2015.01.12. 
Maybe its time to discuss when a next feature release could be cut.

After recent fixes the current NBs are rather stable.
From the first look I do only see these issues as "To Do" before a possible release:
- [x] #4711 - No regression - but glitches in a new feature (Ignore list)
- [x] #4712 - No regression - but glitches in a new feature (Ignore list)
- [x] #4702 - Would be a regression <-- Update: No regression, thus set to done
- [x] #4726 - Need to check if this is a reproducible new issue
- [x] #4686 - Waypoint texts might get mixed up
- [x] #4741 - Search result can not be filtered
- [x] #4756 - Owned caches not shown (Under investigation!)
- [x] #4757 - Add icon for new menu item "Hint"

@rsudev
Does it make sense to wait for a Mapsforge 0.5 implementation or maybe does it make sense to even release before including this to gain more debugging time?

**Comment by rsudev** [comment]
No, it doesn't make sense to wait for the mapsforge 0.5 implementation.
The only feature I right now have in the pipeline (because I was doing a multi today and was really pissed that I had to always scroll down to the save button) is an ActionBar for the EditWaypoint activity with an automatic save on pressing 'back'.
This should be ready in the next two or three days.

**Comment by Lineflyer** [comment]
@rsudev #1536 #3443 
And if you are working on that activity....maybe this would fit as well: #3857 #4402

**Comment by rsudev** [comment]
Thanks for highlighting, I will at least take a look into that direction. And perhaps #4686, even though I have no clear idea how to tackle this.

**Comment by dl1ely** [comment]
@rsudev Thanks god finally is someone is pissed, too :-) :+1:

**Comment by Bananeweizen** [comment]
Sounds okay for me. I've started working on a new filter activity allowing more easy creation of combined filters but I will only continue that in April after Vietnam. there should be a branch in my repo if someone wants to take a look at that before.

**Comment by Lineflyer** [comment]
Set #4702 as done here, as it is no regression. See issue for details.

**Comment by samueltardieu** [comment]
I'll comment out the ignoring capabilities since we have to polish them before including them in the release.

**Comment by Lineflyer** [comment]
I set both ignore list related issues to done here because of the feature removal on release branch.
So we are "green" based on the theory.

RC is building and will be tested.
I think we should give us a longer testing period (unless we are forced to release for some reason) as the last release was just some days ago and users might get annoyed if the release cycle is <1week.

**Comment by Lineflyer** [comment]
Added #4741 to the list of ToDo.

**Comment by Lineflyer** [comment]
Another comment:
I would like your opinion for including #4729 into this upcoming release?!

**Comment by Lineflyer** [comment]
Milestones adapted.
Thanks @samueltardieu for the hint.

**Comment by rsudev** [comment]
Yesterday I issued my implementation for #3443 as a PR - should this go int release or not?

**Comment by samueltardieu** [comment]
I think it would be best to discuss it more, as to avoid heavy infrastructure modifications in `CgeoApplication` and `AbstractActivity` classes if possible.

**Comment by Lineflyer** [comment]
We still need #4741 to be solved before we can release as this is a regression.
Still the RC looks stable for me so I will announce it via Facebook today to get more feedback.
@SammysHP Feel free to do alike on G+ and Twitter?!

**Comment by SammysHP** [comment]
Twitter done, @samueltardieu is responsible for G+.

**Comment by samueltardieu** [comment]
G+ done

**Comment by Bananeweizen** [comment]
Has an icon been added for the hint menu in compass and map?

**Comment by Lineflyer** [comment]
@Bananeweizen No. AFAIR we discussed a suitable one but no one added it yet.

**Comment by rsudev** [comment]
Is the result of the discussion availabel somewhere, so somebody could add it?

**Comment by Lineflyer** [comment]
#4741 fixed.

#4756 added until clarified (Not reproducible right now).

**Comment by Lineflyer** [comment]
@rsudev: Its here: http://irclogs.cgeo.org/?year=2015&month=2&day=11

**Comment by Lineflyer** [comment]
I suggest another download of strings tomorrow noon and release after that.
Tests have been OK over the past days and no negative feedback from support or social network.

**Comment by rsudev** [comment]
I will be on the road until afternoon tomorrow. I also suggest, if time permits, to possibly approve voted strings in the languages where we don't have 'active' proofreaders before this download.

**Comment by Lineflyer** [comment]
@rsudev I have already done this today evening, but we should check and do the same tomorrow noon again.

**Comment by Lineflyer** [comment]
I have proofread all existing and completed spanish myself.
So we are as complete as possible. I will be on the road soon, so if someone is available feel free to import strings from crowdin once more and release to PlayStore.

The current RC has been tested on Android 2.3 and Android 4.4 by myself and looks ok.
Only problem for now: #4772 (The showcase stopped working normal for me....but did this before. No idea why.

Once we are ready (either #4772 is solved or we decide to leave it open for now) we need to do:

- [ ] Import from Crowdin
- [ ] Complete version code in changelog_release.xml
- [ ] Build release
- [ ] Publish to 100% on Play
- [ ] Tag release on Github
- [ ] Upload APK on Github
- [ ] Once available on Play: Remove the notification message and send notification for new release
- [ ] Post to social networks

**Comment by Lineflyer** [comment]
Update posting above as #4772 turned out to be unreproducible (after rebooting my device).

**Comment by Lineflyer** [comment]
Release is uploaded to Play (see above).

**Comment by samueltardieu** [comment]
Tag, notification and G+ done.

**Comment by Lineflyer** [comment]
Binary uploaded to Github
@SammysHP Can you post on Twitter that the new release is out?

**Comment by SammysHP** [comment]
On mobile since Friday morning, posted to Twitter yesterday afternoon.
